t/lib/t/MusicBrainz/Server/Email.pm

```diff
@@ -47,7 +47,7 @@ test all => sub {
     is($delivery->{envelope}->{from}, 'noreply@musicbrainz.org', "Envelope from is noreply@...");
     my $e = $delivery->{email};
     $email->transport->clear_deliveries;
-    is($e->get_header('From'), '"Editor 1" <"Editor 1"@users.musicbrainz.org>', 'Header from is Editor 1 @users.musicbrainz.org');
+    is($e->get_header('From'), '"Editor 1" <noreply@musicbrainz.org>', 'Header from is "Editor 1" <noreply@musicbrainz.org>');
```

Comment:
I'm not sure why we quote the editor name there, but.

Reply:
This is done by the function `format` of the module `Email::Address` to follow RFC 2822.

Reply:
Right, I meant 'cause it doesn't appear to contain any characters that need to be quoted (?), but I guess it just always quotes it.